llm/ext_server/httplib.h

```diff
@@ -23,7 +23,7 @@
 #endif
 
 #ifndef CPPHTTPLIB_CONNECTION_TIMEOUT_SECOND
-#define CPPHTTPLIB_CONNECTION_TIMEOUT_SECOND 300
+#define CPPHTTPLIB_CONNECTION_TIMEOUT_SECOND 600
```

Comment:
After reviewing, we shouldn't change this file as it's an http library

Instead let's change `--timeout` in https://github.com/ollama/ollama/blob/main/llm/server.go#L187